modAionImpl/src/org/aion/zero/impl/db/AionBlockStore.java

```diff
@@ -446,11 +457,18 @@ public void reBranch(AionBlock forkBlock) {
 
                 while (currentLevel > forkBlock.getNumber()) {
 
+                    if (preBranchingBlk == null) {
+                        preBranchingBlk = new Stack<>();
```

Comment:
I would prefer initializing the data structure where it is declared, instead of having so many `if` statements.